src/components/routes/Host/ListingForm/AccommodationsForm/AccommodationsForm.tsx

```diff
@@ -43,6 +43,7 @@ const AccommodationsForm = (props: any): JSX.Element => {
         <InputWrapper>
           <Field
             name="sleepingArrangement"
+            onFocus={() => setFocus('sleepingArrangement')}
```

Comment:
might be a little easier to manage these as enums so you can just change it in one place

Reply:
updated to use `LISTING_FIELDS` enum